### ISSUE TITLE
Event tracker CTA for Cerner Re-direct - Public Website widget

### ISSUE THREAD

**Issue by ldelacosta** [department-of-veterans-affairs/va.gov-team#62996]
## Description
We will need to add an event tracker for the Cerner re-direct link on the Public Websites widget. 

---

## Acceptance Criteria
Background: the `va_online_scheduling_static_landing_page` toggle is on

- [ ] Given when the user is authenticated and is registered at a Cerner site
      When the user is on the Appointments widget on the static landing page and clicks on `Go to My VA Health`
        Then it will trigger an event `vaos-cerner-redirect-static-landing-page` in UA4

### Design Assets
Not applicable 

### Out of scope 
Adding the tracking event to GA4. This will be part of #61355. 

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

---

**Comment by simiadebowale** [comment]
blocked by https://app.zenhub.com/workspaces/appointments-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/61692

**Comment by ldelacosta** [comment]
Could we un-block this ticket? @simiadebowale

**Comment by simiadebowale** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBb0F4IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--6d3ccfd47ec685eeea698c637a01f356df1baa00/Screenshot%202023-08-23%20at%206.27.34%20PM.png)
confirmed that we are sending the event `vaos-cerner-redirect-static-landing-page`
cc: @ldelacosta

**Comment by ldelacosta** [comment]
Event tracker has been confirmed in UA. Closing out the ticket. cc: @JRRoof
